yarn-project/simulator/src/avm/avm_simulator.test.ts

```diff
@@ -140,11 +141,11 @@ describe('AVM simulator: transpiled Noir contracts', () => {
   describe.each([
     ['poseidon_hash', poseidonHash],
     ['pedersen_hash', pedersenHash],
-    ['pedersen_hash_with_index', (m: Buffer[]) => pedersenHash(m, 20)],
-  ])('Hashes with field returned in noir contracts', (name: string, hashFunction: (data: Buffer[]) => Fr) => {
+    ['pedersen_hash_with_index', (m: Fieldable[]) => pedersenHash(m, 20)],
+  ])('Hashes with field returned in noir contracts', (name: string, hashFunction: (data: Fieldable[]) => Fr) => {
     it(`Should execute contract function that performs ${name} hash`, async () => {
       const calldata = [new Fr(1), new Fr(2), new Fr(3)];
-      const hash = hashFunction(calldata.map(f => f.toBuffer()));
+      const hash = hashFunction(calldata);
```

Comment:
These manual conversions are no longer necessary because the hash functions serialize it as they need it.